src/components/POLine/Item/ItemForm.js

```diff
@@ -123,11 +126,24 @@ class ItemForm extends Component {
 
     dispatch(change(fieldName, value));
 
+    this.updateContributor();
```

Comment:
this shouldn't be here, all required info is coming from `getInventoryData` above